main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -70,8 +70,6 @@ object JvmOps {
     case MonoType.Enum(_) => JvmType.Object
     case MonoType.Arrow(_, _) => getFunctionInterfaceType(tpe)
     case MonoType.Native(clazz) => JvmType.Reference(JvmName.ofClass(clazz))
-
-    case _ => throw InternalCompilerException(s"Unexpected type: '$tpe'.", SourceLocation.Unknown)
```

Comment:
This case matches nothing so I deleted it

Reply:
Neat